src/pages/projects.jsx

```diff
@@ -1,109 +1,154 @@
 import React, { useState } from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import Circles from "../../images/circles_0001.png";
 import Cityscape from "../../images/cityscape_0001.png";
 
-const StyledDiv = styled.div`
+const ProjectsContainer = styled.div`
     margin: 0 70px 25px 70px;
 
     @media screen and (max-width: 900px), (max-height: 640px) {
         margin: 0 20px 25px 20px;
     }
 `;
 
-const StyledH1 = styled.h1`
-    font-size: 50px;
-    margin-bottom: 20px;
-`;
+const Header = styled.h1`
+    ${(props) =>
+        props.main &&
+        css`
+            font-size: 70px;
+            margin-bottom: 20px;
+
+            @media screen and (max-width: 900px), (max-height: 640px) {
+                font-size: 50px;
+            }
+        `}
 
-const PageOverviewStyling = styled.p`
-    font-size: 25px;
-    margin-bottom: 100px;
+    ${(props) =>
+        props.project &&
+        css`
+            font-size: 55px;
+            margin-bottom: 20px;
+
+            @media screen and (max-width: 900px), (max-height: 640px) {
+                font-size: 35px;
+            }
+        `}
 `;
 
-const ProjectDescriptionStyling = styled.p`
-    font-size: 25px;
-    margin-bottom: 20px;
+const Description = styled.p`
+    ${(props) =>
+        props.main &&
+        css`
+            font-size: 40px;
+            margin-bottom: 80px;
+            line-height: 1.5;
+
+            @media screen and (max-width: 900px), (max-height: 640px) {
+                font-size: 25px;
+            }
+        `}
+
+    ${(props) =>
+        props.project &&

```

Comment:
Instead of having two props, `main` and `project`, why not just have one prop, and if it's `true` do one thing, otherwise (so if it's `false`) do the other thing? It makes the code cleaner and reduces complexity.

Reply:
I'm saying you're currently using 2 booleans when you only need 1.

It sounds like you're saying you want to use a string instead of a boolean since you plan to have multiple options in the future, which is fine, but just don't use 2 booleans when you only need 1 like you're doing in this PR.

Reply:
Right now you're using a short-circuit (the `&&`) to render:
```
css`
    font-size: 40px;
    margin-bottom: 80px;
    line-height: 1.5;
    @media screen and (max-width: 900px), (max-height: 640px) {
        font-size: 25px;
    }
`
```
if `props.main` is true, and to render nothing if `props.main` is false. Let's call this snippet `A`. Then in the same styled component, you're using another short-circuit to render:
```
css`
    font-size: 30px;
    margin-bottom: 20px;
    line-height: 1.5;
    @media screen and (max-width: 900px), (max-height: 640px) {
        font-size: 20px;
    }
`
```
if `props.project` is true, and to render nothing if `props.project` is false. Let's call this snippet `B`. Instead, you should render the css for `props.main` when `props.main` is true, and render the css for `props.project` when `props.main` is false. One way you can do this is by using a ternary instead of a short-circuit: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

So in summary, what you're currently doing:
Using a short-circuit to render `A` or nothing
Then using a short-circuit to render `B` or nothing

What you should be doing:
Using a ternary to render `A` or `B`

Also note that your `Header` styled component has this same issue.

Reply:
But if you want to use a string instead of a boolean for `props.main`, instead of a ternary you can use a switch statement: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

In which case I think a name like `props.layoutType` or something might be better. Then you can use `props.layoutType` as your switch expression, and then choose what css to render in the case statements.

Reply:
But now that I think of it, if your styled component solely uses one case of css out of many, it makes a lot more sense to just have a separate styled component for each case of css. Having branching logic to decide what the entire return value is for the styled component doesn't make any sense when you could just make a separate styled component instead.

Reply:
I think I see what you're saying! Thank you very much for explaining and showing the alternatives! I decided to separate them all into styled components to make it cleaner.